src/PSPackageProject.psd1

```diff
@@ -74,7 +74,8 @@ RequiredModules = @(
 
 # Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
 FunctionsToExport = @(
-    'New-PSPackageProjectHelp', 'Invoke-PSPackageProjectTest'
+    'New-PSPackageProjectHelp'
```

Comment:
Do the new functions need to be exported here?

Reply:
no